Samples/SDKFeatures/SDKFeatures/ViewControllers/ActionPlans/MHVActionPlansListViewController.m

```diff
@@ -0,0 +1,187 @@
+    metrics.scheduledDays = @[@"Monday", @"Wednesday", @"Friday"];
+    metrics.occurrenceCount = [NSNumber numberWithInt:1];
+    metrics.windowType = @"Daily";
+    
+    MHVActionPlanTask *frequencyTask = [[MHVActionPlanTask alloc] init];
+    frequencyTask.name = @"Do a fun activity.";
+    frequencyTask.shortDescription = @"Do an activity to get some exercise.";
+    frequencyTask.longDescription = @"Go for a run, hike a mountain, ride your bike around town, or something else to get moving.";
+    frequencyTask.imageUrl = @"https://img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE1rXx2?ver=d68e";
+    frequencyTask.thumbnailImageUrl = @"https://img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE1s2KS?ver=0ad8";
+    frequencyTask.taskType = @"Other";
+    frequencyTask.signupName = @"Do a fun activity.";
+    frequencyTask.associatedObjectiveIds = [[NSArray alloc] initWithObjects:objective._id, nil];
+    frequencyTask.trackingPolicy = policy;
+    frequencyTask.completionType = @"Frequency";
+    frequencyTask.frequencyTaskCompletionMetrics = metrics;
+    
+    MHVActionPlan *newPlan = [[MHVActionPlan alloc] init];
+    NSNumber *rand = [NSNumber numberWithUnsignedInteger:arc4random_uniform(100)];
+    newPlan.name = [NSString stringWithFormat:@"My new plan (%@)", rand];
+    newPlan._description = @"A sample activity plan";
+    newPlan.imageUrl = @"https://img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE10omP?ver=59cf";
+    newPlan.thumbnailImageUrl = @"https://img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE10omP?ver=59cf";
+    newPlan.category = @"Activity";
+    newPlan.objectives = [[NSArray<MHVObjective> alloc] initWithObjects:objective, nil];
+    newPlan.associatedTasks = [[NSArray<MHVActionPlanTask> alloc] initWithObjects:frequencyTask, nil];
+    
+    [_connection.remoteMonitoringClient createActionPlanWithActionPlan:newPlan completion:^(MHVSystemObject * _Nullable output, NSError * _Nullable error) {
+        [[NSOperationQueue mainQueue] addOperationWithBlock:^
+         {
+             if (!error)
+             {
+                 [self loadActionPlans];
+             }
+         }];
+    }];
+}
+
+// -------------------------------------
+//
+// UITableViewDataSource & Delegate
+//
+// -------------------------------------
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
+{
+    return [_actionPlans count];
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    UITableViewCell *cell = [self.tableView dequeueReusableCellWithIdentifier:@"MHVCell"];
+    
+    if (!cell)
+    {
+        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"MHVCell"];
+    }
+    
+    NSArray *keys = [self.actionPlans allKeys];
+    
+    NSString *typeName = keys[indexPath.row];
```

Comment:
allKeys & allValues on NSDictionary doesn't return the keys or values in any order.  I would do an array of MHVActionPlanInstance's, then lookup the name and identifier on that object here and in didSelectRowAtIndexPath:  Otherwise will get weird mismatches if tap plan A and could be shown plan B